### PR TITLE
feat: show hardware used in tests in tree details header

### LINKED ISSUE #384: Tree Details Page: show hardware used for testing in the page header

### PR DESCRIPTION
**Pull request by anajalvarenga**
Adding Hardware Used table in Tree details header

image

The table data shouldn't change by adding filters
- closes #384

### CODE REVIEW

**Comment by Francisco2002** [comment]
very nice!!

In the page, the height of loading content is to big comparing with the card height. Make then the same height

Screenshot from 2024-10-23 09-26-41
Screenshot from 2024-10-23 09-26-55

**Comment by anajalvarenga** [comment]
> very nice!!
> 
> In the page, the height of loading content is to big comparing with the card height. Make then the same height
> 
> Screenshot from 2024-10-23 09-26-41 Screenshot from 2024-10-23 09-26-55

Since I don't have the data when the loading is showing, I can't fit it to the same size as the card, but I reduce its height

**Comment by WilsonNet** [comment]
@anajalvarenga just put an average height from something that has 2 lines (there is a prop in the query switcher for that)

**Review by WilsonNet — APPROVED** [review]
great job

File: dashboard/src/pages/TreeDetails/Tabs/TestCards.tsx [hunk 1]
```diff
@@ -294,3 +296,36 @@ const IssuesList = ({ issues, title }: IIssuesList): JSX.Element => {
...
+            return (
+              <Badge
+                key={hardware}
+                variant="outline"
+                className="text-sm font-normal"
+              >
+                {hardware}
+              </Badge>
+            );
+          })}
```
Comment (lfjnascimento) [inline dashboard/src/pages/TreeDetails/Tabs/TestCards.tsx:325]: put this block inside the `useMemo`

**Review by lfjnascimento — APPROVED** [review]

**Pull request merged**